lib/charms/postgresql_k8s/v1/postgresql.py

```diff
@@ -242,11 +243,11 @@ def create_database(
             if cursor.fetchone() is None:
                 cursor.execute(SQL("SET ROLE charmed_databases_owner;"))
                 cursor.execute(SQL("CREATE DATABASE {};").format(Identifier(database)))
-            cursor.execute(
-                SQL("REVOKE ALL PRIVILEGES ON DATABASE {} FROM PUBLIC;").format(
-                    Identifier(database)
+                cursor.execute(
+                    SQL("REVOKE ALL PRIVILEGES ON DATABASE {} FROM PUBLIC;").format(
+                        Identifier(database)
+                    )
                 )
```

Comment:
Execute this `REVOKE` operation only when the database is created, because it's not needed later, as it's already done and would cause an error when a relation requesting the same database name is established.